### DISCUSSION TITLE
Add support for Anthropic Bedrock SDK

### DISCUSSION THREAD

**Discussion by shreecodes** [traceloop/openllmetry-js#361] [Ideas]
Would love for this library to support Anthropic's Bedrock SDK, via the `instrumentModules` initialization option. 

https://github.com/anthropics/anthropic-sdk-typescript/tree/main/packages/bedrock-sdk

**Comment by nirga** [comment]
Sorry @shreecodes for missing this! We already support bedrock, is there something specific you're missing?
